content/docs/user-guide/related-technologies.md

```diff
@@ -82,8 +82,8 @@ _Luigi_, etc.
 
 ## Experiment management software
 
-- DVC uses Git as the underlying layer for data, pipelines, an experiment
-  versioning, instead of a custom web application.
+- DVC integrates with Git, so it remains the underlying version control tool for
+  data, pipelines, an experiments, instead of reinventing the wheel.
```

Comment:
this gives it a quite different meaning

Reply:
Yes for 2 reasons.
1. A correction: DVC doesn't use Git, it integrates with Git.
2. "instead of a custom web application"? It's not clear why we're talking about web apps here IMO. I guess in reference to MLFlow's interface? Any other examples? Maybe we should mention them directly like we mention git-lfs for ex.
  In any case, "uses Git for versioning instead of a web app" still doesn't make much sense. Are web apps used for versioning? But maybe I'm missing something or not understanding the original intention of this text.

Reply:
For now I'm simplifying it to:
```md
- DVC integrates with Git as the underlying version control tool for data,
  pipelines, an experiments.
```

Reply:
yes, I was referring to the `instead of reinventing the wheel.` part mostly (integrates or uses - not that important, uses sounds friendlier though)


`a custom web application` is not the best either, agreed. What it was trying to say that `DVC saves info into Git vs some external app or external database or external API`

Reply:
OK changed back to "uses" although it's slightly incorrect... But you're right, easier to read/ friendlier. I also updated the other part of the explanation. PTAL